### ISSUE TITLE
> ![image](https://user-images.githubusercontent.com/26559584/79293390-3392a000-7f06-11ea-9531-8afd38fc962c.png)

### ISSUE THREAD

**Issue by zheng-2019** [zylo117/Yet-Another-EfficientDet-Pytorch#81]
> ![image](https://user-images.githubusercontent.com/26559584/79293390-3392a000-7f06-11ea-9531-8afd38fc962c.png)
> i train custom data content 1 class，but the loss can't turn small
> @zylo117 and resume training is not normal,when i resume train,the loss beacma very large

it's expoding gradients. try 100 times smaller lr

_Originally posted by @zylo117 in https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/issues/66#issuecomment-613785254_

set lr = 1e-2 same normal,i try again

**Comment by zylo117** [comment]
so... the original lr is 1? are you serious?

try 1e-5...

and pls, read and run the tutorial

**Comment by zheng-2019** [comment]
when train as:python train.py -c 2 --batch_size 8 --lr 1e-5 --num_epochs 10 \  --load_weights /path/to/your/weights/efficientdet-d2.pth \  --head_only Truethe loss not turn smallerbut i set --lr 1e-2,it's same normal as :






------------------&nbsp;原始邮件&nbsp;------------------
发件人: "zylo117"<notifications@github.com&gt;; 
发送时间: 2020年4月15日(星期三) 上午10:58
收件人: "zylo117/Yet-Another-EfficientDet-Pytorch"<Yet-Another-EfficientDet-Pytorch@noreply.github.com&gt;; 
抄送: "郑江红"<526078586@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [zylo117/Yet-Another-EfficientDet-Pytorch] &gt; ![image](https://user-images.githubusercontent.com/26559584/79293390-3392a000-7f06-11ea-9531-8afd38fc962c.png) (#81)





 
so... the original lr is 1? are you serious?
 
try 1e-5...
 
and pls, read and run the tutorial

**Comment by zylo117** [comment]
sorry, can't quite get what you said.

But I know what's going on now.

you can't load a d0 weights into a d2 network.

pls read and run the tutorial
